### PR TITLE
Prep for release of sample metadata tables in reports

### PR DESCRIPTION
**Pull request by allyhawkins**
## Purpose

Here I'm adding a changelog to mention that reports now include the sample metadata tables. There are no other docs changes with this release. We also have quite a few changes happening soon and I'm not sure which order they are going to happen in, so to avoid future conflicts, I created this feature branch off `main` to add in this change log. Once we are ready to release those updated reports, we just need to add a date to the changelog and continue with the release. Because of that I'm filing this as a draft right now. 

### Issue addressed

#325 

## Checklist for publishing to `main`

- [x] This branch contains all commits/content ready to be released.
- [ ] The most recent CHANGELOG entry includes a date.
If this release required a CHANGELOG entry, it should be today's date.

### CODE REVIEW

**Comment by allyhawkins** [comment]
Since we plan on releasing this next, I have added the changes from #329 and #335 to this branch so they will be included in the next release. 

I still need review of the changelog, so I'm going to request review of that now and then will request another review once we are ready to add in the date and actually merge these changes in.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## PLACEHOLDER FOR METADATA ADDITION IN REPORTS
+
+* A table containing sample metadata (e.g., age, sex, diagnosis) is now available in both the QC report (`qc.html`) and the supplemental cell type report (`celltype-report.html`) included in all downloads.
```
Comment (sjspielman) [inline docs/CHANGELOG.md:17]: I added `etc.` to be a little clear that there are more than those three. Is it too much with eg and etc?

```suggestion
* A table containing sample metadata (e.g., age, sex, diagnosis, etc.) is now available in both the QC report (`qc.html`) and the supplemental cell type report (`celltype-report.html`) included in all downloads.
```

**Review by sjspielman — COMMENTED** [review]
Everything looks good to me here at this stage! One small changelog comment at your discretion.

File: docs/CHANGELOG.md (hunk 1 above)
Comment (jaclyn-taroni) [reply docs/CHANGELOG.md:17]: I don't think you need `etc.` if you are listing examples.

**Pull request merged**